.buildkite/pipeline.yml

```diff
@@ -71,6 +71,7 @@ steps:
       buildkite-agent artifact download "*.exe" .
 
       echo "--- :node: Generating Release Manifest"
+      export IS_DEV_BUILD=true
```

Comment:
Or `IS_DEV_BUILD=true node ./scripts/generate-releases-manifest.mjs` to make it explicit the flag is specifically for the `generate-releases-manifest` script.